examples/0_basic/0_create_did.rs

```diff
@@ -1,54 +1,48 @@
-// Copyright 2020-2022 IOTA Stiftung
-// SPDX-License-Identifier: Apache-2.0
-
-use anyhow::Context;
-use examples::get_address_with_funds;
-use examples::random_stronghold_path;
-use examples::NETWORK_ENDPOINT;
-use identity_core::convert::ToJson;
-use identity_core::crypto::KeyPair;
-use identity_core::crypto::KeyType;
-use identity_did::verification::MethodScope;
-use identity_stardust::NetworkName;
-use identity_stardust::StardustClientExt;
-use identity_stardust::StardustDocument;
-use identity_stardust::StardustIdentityClientExt;
-use identity_stardust::StardustVerificationMethod;
+use identity_iota::core::ToJson;
```

Comment:
Add back the licence header. I don't think it's necessary for the examples, but we should be consistent.

Fixing #921 would have caught this.

```suggestion
// Copyright 2020-2022 IOTA Stiftung
// SPDX-License-Identifier: Apache-2.0

use identity_iota::core::ToJson;
```